Odata-docs/client/grouping-and-aggregation.md

```diff
@@ -0,0 +1,481 @@
+---
+title: "Grouping and Aggregation in OData Client"
+description: "This tutorial describes how to group and aggregate data in Odata Client"
+
+author: gathogojr
+ms.author: jogathog
+ms.date: 7/25/2022
+ms.topic: article
+ 
+---
+# Grouping and Aggregation in OData Client
+**Applies To**: [!INCLUDE[appliesto-odataclient](../includes/appliesto-odataclient-v7.12.1.md)]
+
+OData Client [v7.12.1](https://www.nuget.org/packages/Microsoft.OData.Client/) introduced support for grouping and aggregation. In this page, we look at the various supported scenarios. The sample data from the 
+[OASIS OData Extension for Data Aggregation Version 4.0](http://docs.oasis-open.org/odata/odata-data-aggregation-ext/v4.0/cs01/odata-data-aggregation-ext-v4.0-cs01.html#_Toc378326287) will be used to illustrate the scenarios supported by this feature.
+
+## Simple Aggregation
+The standard LINQ aggregation methods include [Average](/dotnet/api/system.linq.queryable.average), [Sum](/dotnet/api/system.linq.queryable.sum), [Min](/dotnet/api/system.linq.queryable.min), and [Max](/dotnet/api/system.linq.queryable.max). We support overloads of these methods that accept two parameters; a sequence of values - `source`: `IQueryable<TSource>`, and a projection function to apply to each element - `selector`.
+
+These methods are translated into the corresponding OData URL.
+
+### Standard LINQ Aggregation Methods Examples
+1. Return average of sale amounts
```

Comment:
I think this should be subheadings so that they appear in the generated table of contents and are easier to navigate to.

Reply:
Actually, I'm not sure if the generate list of sections enumerates anything beyond level-2 headings. This is what it currently looks like:

![image](https://user-images.githubusercontent.com/8460169/182409429-68ddf625-2637-4eab-ac97-b5eb6d16ba57.png)